docs/management/debugging.md

```diff
@@ -35,82 +35,82 @@ GDB can be used in two ways:
 * It can attach to a running program and inspect the state of it at runtime.
 * It can inspect the state of a program that already terminated using what is called a *core file*, that is, the image of the memory at the time the program was running.
 
-From the point of view of investigating Redis bugs we need to use both of these
-GDB modes. The user able to reproduce the bug attaches GDB to their running Redis
+From the point of view of investigating Valkey bugs we need to use both of these
+GDB modes. The user able to reproduce the bug attaches GDB to their running Valkey
 instance, and when the crash happens, they create the `core` file that in turn
-the developer will use to inspect the Redis internals at the time of the crash.
+the developer will use to inspect the Valkey internals at the time of the crash.
 
 This way the developer can perform all the inspections in his or her computer
-without the help of the user, and the user is free to restart Redis in their
+without the help of the user, and the user is free to restart Valkey in their
 production environment.
 
-## Compiling Redis without optimizations
+## Compiling Valkey without optimizations
 
-By default Redis is compiled with the `-O2` switch, this means that compiler
-optimizations are enabled. This makes the Redis executable faster, but at the
-same time it makes Redis (like any other program) harder to inspect using GDB.
+By default Valkey is compiled with the `-O2` switch, this means that compiler
```

Comment:
(This is actually not true, but we can fix that another time.) It's good that we do all rebranding first.